crates/release_plz_core/src/next_ver.rs

```diff
@@ -578,24 +591,32 @@ fn get_changelog(
     Ok(changelog)
 }
 
+fn get_package_path(
+    package: &Package,
+    repository: &Repo,
+    project_root: &Path,
+) -> anyhow::Result<PathBuf> {
+    let package_path = package.package_path()?;
+    let relative_path = package_path
+        .strip_prefix(project_root)
+        .context("error while retrieving package_path: project root not found")?;
+    let result_path = repository.directory().join(relative_path);
+
+    Ok(result_path)
+}
+
 #[instrument(
     skip_all,
     fields(package = %package.name)
 )]
 fn get_diff(
     package: &Package,
-    run_semver_check: RunSemverCheck,
     registry_packages: &PackagesCollection,
     repository: &Repo,
     project_root: &Path,
 ) -> anyhow::Result<Diff> {
-    let package_path = {
-        let relative_path = package
-            .package_path()?
-            .strip_prefix(project_root)
-            .context("error while retrieving package_path")?;
-        repository.directory().join(relative_path)
-    };
+    let package_path = get_package_path(package, repository, project_root).unwrap();
```

Comment:
```suggestion
    let package_path = get_package_path(package, repository, project_root)?;
```
let's return the error to the caller instead of unwrap 👍